doc/SONiC_202012_Release_Notes.md

```diff
@@ -0,0 +1,143 @@
+
+#### Multi-DB enhancement-Part 2
+**Pull Requests** : [5773](https://github.com/Azure/sonic-buildimage/pull/5773) & [1205](https://github.com/Azure/sonic-utilities/pull/1205)
+
+#### ONIE FW tools
+SONiC FW utility uses platform API to interact with the various platform components. SONiC FW utility extends to support for the automatic firmware update based on "platform_components.json" under platform directory and next reboot option which is passed as a option for fwutil update all fw command. SONiC FW utility also extends to support for the automatic firmware update with a custom firmware package that can include any firmware update tool and the firmware update tool will be used for the firmware update if it's specified in the "platform_components.json".
+**Pull Requests** : [1165](https://github.com/Azure/sonic-utilities/pull/1165), [106](https://github.com/Azure/sonic-platform-common/pull/106)
+
+#### PDDF advance to SONiC Platform 2.0, BMC
+SONiC OS is portable across different network devices with supported ASIC via Switch Abstraction Interface (SAI). These devices primarily differ in the way various device specific hardware components are accessed, and thus require custom device drivers and python plugins. Each platform vendor implements these custom device drivers and plugins. The feature requirement is to support a SONiC platform driver development framework to enable rapid development of custom device drivers and plugins.
```

Comment:
Please say as follows: -

PDDF is a rapid platform development environment for SONiC, enabling new platforms to be quickly integrated into the SONiC eco-system by automating much of the routine development using simple description files. it supports both the SONiC Platform 1.0 and 2.0 designs (though 2.0 is preferred). Also supports BMC.

Reply:
Addressed